addons/test_event_full/tests/test_performance.py

```diff
@@ -444,15 +444,15 @@ def test_events_browse_event_manager(self):
         # website customer data
         with freeze_time(self.reference_now):
             self.authenticate('user_eventmanager', 'user_eventmanager')
-            with self.assertQueryCount(default=50):  # tef only: 49
+            with self.assertQueryCount(default=60):  # com runbot: 59 - ent runbot:60
```

Comment:
Can I ask why this adds 10 and then 9 requests?
Is it because of the added `track="1"` on the template alone?

If so, would be great to explain that in the commit message.

Reply:
Actually, that was the question I was asking in the task because when running the test locally I get crazy number (10 times higher) :-)

As the only thing we do here is to add an header that can potentially decrease the number of queries (no tracking for some pages) and add the track="1" on event that add some queries (to track the page: visitor, ...).
I think we can safely assume that the added queries are due to the tracking of that additional page.
I have just taken the number from the runbot report.
I will add a comment in the commit explaining the added page to track increase the number of queries in those tests.

I have investigate why I don't get the same value locally as in the runbot but failed until now. Here is what I have found:

The 2 tests are annotated with the warmup decorator which run test first without checking the assertion and then a second time, this time checking the assertion.
The decorator restore the DB between the 2 tests.
I suppose the idea is that the second time there will be no prefetch of the URL in the page because the browser cache will be filled.
It works on the runbot but not locally. It is like the browser cache is cleaned between the 2 run locally and not in the runbot but I don't see why (library version ? No demo data ?).

If I do manually what warmup do but without restoring DB, invalidate cache, ..., I get similar result as in the runbot:

test_events_browse_event_manager
First time: 475
Second time: 56

test_events_browse_public
First time: 359
Second time: 39

Reply:
As suggested, I have pushed the code without track=1 for the "event" page and the rest unchanged and I get the same number as before so it shows that  he added queries come from the added track=1 attribute for the "event" page.

Without track=1 on the page /event:
test_events_browse_event_manager: # com runbot: 49 - ent runbot:50
test_events_browse_public: # com runbot: 30 - ent runbot: 30

Reply:
Ok so we know it's because of that change then, thanks for checking in-depth!